website/docs/en/config/dev/watch-files.mdx

```diff
@@ -0,0 +1,52 @@
+# dev.watchFiles
+
+- **Type:**
+
+```ts
+type WatchFiles =
+  | string
+  | string[]
+  | {
+      paths: string | string[];
+      // watch options for chokidar
+      options?: WatchOptions;
+    };
+```
+
+- **Default:** `undefined`
+
+Watch files and directories for changes. When a file changes, the page will be reloaded.
+
+If `dev.hmr` or `dev.reload` is set to false, `watchFiles` will be ignored.
```

Comment:
- `dev.reload` -> `dev.liveReload`
- If `dev.hmr` is false, the reload can still work, so can we still support watch files in this case?

```suggestion
If `dev.liveReload` is set to false, `watchFiles` will be ignored.
```

Reply:
```
if (dev.hmr || dev.liveReload) {
    watcher.on('change', () => {
      if (compileMiddlewareAPI) {
        compileMiddlewareAPI.sockWrite('static-changed');
      }
    });
  }
  ```
  I have it set so that when either `hmr` or `liveReload` is set to false, `watchFiles` is disabled. 
  Do you mean to change it so that `hmr` and `liveReload` are both false?
  ```
    if (dev.hmr && dev.liveReload) {
    watcher.on('change', () => {
      if (compileMiddlewareAPI) {
        compileMiddlewareAPI.sockWrite('static-changed');
      }
    });
  }
  ```

Reply:
No, I'm making the document consistent with the code, not intending to change the code logic.

Reply:
`dev.hmr || dev.liveReload` is same as "If both `dev.hmr` and `dev.liveReload` are set to false, `watchFiles` will be ignored."

Reply:
Oops! I misunderstood it as the opposite meaning.😅

Reply:
haha never mind